pybamm/models/submodels/thermal/base_thermal.py

```diff
@@ -179,32 +179,71 @@ def _get_standard_coupled_variables(self, variables):
-        Q_ohm_vol_av = self._yz_average(Q_ohm_av)
-        Q_rxn_vol_av = self._yz_average(Q_rxn_av)
-        Q_rev_vol_av = self._yz_average(Q_rev_av)
-        Q_vol_av = self._yz_average(Q_av)
+        # Compute the integrated heat source per unit simulated electrode-pair area
+        # in W.m-2. Note: this can still be a function of y and z for
+        # higher-dimensional pouch cell models
+        Q_ohm_Wm2 = Q_ohm_av * param.L
+        Q_rxn_Wm2 = Q_rxn_av * param.L
+        Q_rev_Wm2 = Q_rev_av * param.L
+        Q_Wm2 = Q_av * param.L
+        # Now average over the electrode height and width
+        Q_ohm_Wm2_av = self._yz_average(Q_ohm_Wm2)
+        Q_rxn_Wm2_av = self._yz_average(Q_rxn_Wm2)
+        Q_rev_Wm2_av = self._yz_average(Q_rev_Wm2)
+        Q_Wm2_av = self._yz_average(Q_Wm2)
+
+        # Compute total heat source terms (in W) over the *entire cell volume*, not
+        # the product of electrode height * electrode width * electrode stack thickness
+        # Note: we multiply by the number of electrode pairs, since the Q_xx_Wm2_av
+        # variables are per electrode pair
+        n_elec = param.n_electrodes_parallel
+        A = param.L_y * param.L_z  # *modelled* electrode area
+        Q_ohm_tot = Q_ohm_Wm2_av * n_elec * A
+        Q_rxn_tot = Q_rxn_Wm2_av * n_elec * A
+        Q_rev_tot = Q_rev_Wm2_av * n_elec * A
+        Q_vol_tot = Q_Wm2_av * n_elec * A
+
+        # Compute volume-averaged heat source terms over the *entire cell volume*, not
+        # the product of electrode height * electrode width * electrode stack thickness
+        V = param.V_cell  # *actual* cell volume
+        Q_ohm_vol_av = Q_ohm_tot / V
+        Q_rxn_vol_av = Q_rxn_tot / V
+        Q_rev_vol_av = Q_rev_tot / V
+        Q_vol_av = Q_vol_tot / V
 
         variables.update(
             {
                 "Ohmic heating [W.m-3]": Q_ohm,
                 "X-averaged Ohmic heating [W.m-3]": Q_ohm_av,
                 "Volume-averaged Ohmic heating [W.m-3]": Q_ohm_vol_av,
+                "Integrated Ohmic heating per unit electrode-pair area "
+                "[W.m-2]": Q_ohm_Wm2,
+                "Total Ohmic heating [W]": Q_ohm_tot,
                 "Irreversible electrochemical heating [W.m-3]": Q_rxn,
                 "X-averaged irreversible electrochemical heating [W.m-3]": Q_rxn_av,
                 "Volume-averaged irreversible electrochemical heating "
                 + "[W.m-3]": Q_rxn_vol_av,
+                "Integrated irreversible electrochemical heating per unit "
+                + "electrode-pair area [W.m-2]": Q_rxn_Wm2,
+                "Total irreversible electrochemical heating [W]": Q_rxn_tot,
                 "Reversible heating [W.m-3]": Q_rev,
                 "X-averaged reversible heating [W.m-3]": Q_rev_av,
                 "Volume-averaged reversible heating [W.m-3]": Q_rev_vol_av,
+                "Integrated reversible heating per unit electrode-pair area "
```

Comment:
I avoided "total" as we already use that to mean "sum of irreversible and reversible", but happy to take suggestions

Reply:
I'd suggest that you just delete the word "Integrated"; I agree with @brosaplanella that "per __" intrinsically makes clear how the quantity is scaled. You have long established the approach of using the same name with different units to account for normalisation, and I think that is also fine.

If you are always using "Total" to mean "sum of irreversible and irreversible" then the name "Total reversible heating" is malformed, though.

In the realm of breaking changes, I would call the [W.m-3] quantities a "volumetric heat source" and the [W] quantities a "heat rate".

Also (pedantically), "ohmic" should be lowercase (c.f. voltaic, faradaic). Not worth changing if breaking.

Reply:
Thanks, yes, those “Total…” shouldn’t be there, not sure why I put them there. Probably not worth changing the capitalisation of Ohmic. 

To avoid any other breaking changes I’ll keep the same names and infer things from units.

Reply:
I remember now, the “Total” came from copy the definition of the existing “Total heating [W]” variable. I’ll make these changes. Otherwise, does this look correct?